tests/test_data_arrays.py

```diff
@@ -41,6 +42,9 @@
 ]
 FIELDS = ("Ex", "Ey", "Ez", "Hz")
 INTERVAL = 2
+ORDERS_X = list(np.linspace(-1, 1, 3))
```

Comment:
probably no need to change, but just FYI you can do `np.arange(-1, 2)` instead of `np.linspace(-1, 1, 3)` and it will give you `int` instead of `float` and there's no need to supply the number of points (3).

Reply:
Good point, thanks